components/user-mgt/org.wso2.carbon.identity.user.profile.ui/src/main/resources/web/userprofile/index.jsp

```diff
@@ -178,14 +179,15 @@
                  }
         	     CARBON.showConfirmationDialog("<fmt:message key='remove.message1'/>"+ profile +"<fmt:message key='remove.message2'/>",
                     function() {
-              	       location.href ="remove-profile.jsp?username="+URLEncoder.encode(username)+"&profile="+profile+"&fromUserMgt=<%=fromUserMgt%>";
+              	       location.href ="remove-profile.jsp?username="+username+"&profile="+profile+
+                               "&fromUserMgt=<%=Encode.forJavaScriptBlock(Encode.forUriComponent(fromUserMgt))%>";
```

Comment:
Simply encode username, profile and fromUserMgt variables here for uri. Also apply formatting